207 Course Schedule Medium.java

```diff
@@ -0,0 +1,50 @@
+//207 Course Schedule Medium.java
+//Time Complexity: O(n^2);

```

Comment:
Time Complexity:O(2⋅∣E∣+∣V∣) where ∣V∣ is the number of courses, and ∣E∣ is the number of dependencies. -Leetcode